archway/assetlist.json

```diff
@@ -43,6 +43,66 @@
       },
       "type_asset": "sdk.coin"
     },
+    {
+      "description": "BackBone Labs Liquid Staked Archway",
+      "type_asset": "cw20",
+      "address": "archway17e3z7cr6d29uj54k4ucs6dtqvlxsxhwnwzetqgnp6zeu40cvq7dsd8q60x",
+      "denom_units": [
+        {
+          "denom": "archway17e3z7cr6d29uj54k4ucs6dtqvlxsxhwnwzetqgnp6zeu40cvq7dsd8q60x",
+          "exponent": 0
+        },
+        {
+          "denom": "bARCH",
+          "exponent": 18
+        }
+      ],
+      "base": "archway17e3z7cr6d29uj54k4ucs6dtqvlxsxhwnwzetqgnp6zeu40cvq7dsd8q60x",
+      "name": "BackBone Labs Liquid Staked Archway",
```

Comment:
Please define the relationship to ARCH (via trace type: liquid-stake)

Reply:
do you have an example of this?, i could not find any example of a trace with such type in the repo.
thank you

Reply:
I realize the many Backbone Labs assets don't yet have this defined (but they should, IMO) You can see an example in Stride's assetlist: https://github.com/cosmos/chain-registry/blob/master/stride/assetlist.json#L59

Reply:
added the trace to all the assets. to follow the assets rules for chain-registry.
thank you for the help :D